src/styles/main.scss

```diff
@@ -10,17 +13,59 @@ body {
   font-weight: 900;
 }
 
+.game-field {
+  background: #bbada0;
+  border-spacing: 10px;
+  border-radius: 5px;
+  position: relative;
+  width: 350px;
+  height: 350px;
+}
+
 .field-cell {
+  position: absolute;
   background: #d6cdc4;
-  width: 75px;
-  height: 75px;
+  width: $cell-size;
+  height: $cell-size;
   border-radius: 5px;
   color: #776e65;
   box-sizing: border-box;
   text-align: center;
   vertical-align: center;
```

Comment:
vertical-align: center is not a valid value. It should be middle instead of center.